web_poet/serialization/functions.py

```diff
@@ -140,3 +140,16 @@ def _deserialize_HttpClient(
 
 
 register_serialization(_serialize_HttpClient, _deserialize_HttpClient)
+
+
+def _serialize_dict(o: dict) -> SerializedLeafData:
```

Comment:
I think earier we went away from providing serialize functions for native classes like dict or str. Would it make sense to have it specific to PageParams?

Reply:
We went away from providing them "just in case", but here it's useful for PageParams and I don't think a specialized function is better?

Reply:
I guess my motivation was this: we don't want to accidentally introduce raw dict as a dependency, so that's good if it's explicitly unsupported, and that we don't suggest that it might be supported. That's why if it is the same amount of code, I prefer using PageParams explicitly, not having support for dict. 

But no strong opinon :)